Block/Menu.php

```diff
@@ -84,20 +84,33 @@ protected function getCacheLifetime()
     /**
      * @return \Snowdog\Menu\Model\Menu
      */
-    public function getMenu()
+    protected function loadMenu()
```

Comment:
Is there any reason why it is `protected` instead of `private`?

Reply:
No, I changed to private